src/StockMarket/BuyingAndSelling.tsx

```diff
@@ -51,7 +51,8 @@ export function buyStock(
   // TODO: Validation code in this function and other ones (sellStock, shortStock, sellShort) are bad. I'll rewrite them later.
   if (stock == null || isNaN(shares)) {
     if (ctx) {
-      helpers.log(ctx, () => `Invalid arguments: stock='${stock != null ? stock.name : stock}' shares='${shares}'`);
+      // eslint-disable-next-line @typescript-eslint/no-base-to-string
```

Comment:
Careful, this lint was raising an error correctly. You are interpolating a stock Object (of class Stock), not a symbol. The Stock class does not have a toString implementation. This linter rule is meant to catch that.

I vote for keeping the unsuppressed linter behavior and simply adding toString to Stock.